python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -2649,6 +2649,13 @@ router bgp {{ router_bgp.as }}
 {%         if vrf.router_id is arista.avd.defined %}
       router-id {{ vrf.router_id }}
 {%         endif %}
+{%         if vrf.maximum_paths.paths is arista.avd.defined %}
+{%             set paths_cli = "maximum-paths " ~ vrf.maximum_paths.paths %}
+{%             if vrf.maximum_paths.ecmp is arista.avd.defined %}
+{%                 set paths_cli = paths_cli ~ " ecmp " ~ vrf.maximum_paths.ecmp %}
+{%             endif %}
+      {{ paths_cli }}
+{%         endif %}
```

Comment:
According to CLI `maximum-paths` should come in between `timer bgp` and `bgp additional-paths install`

```
      update wait-install
      timers bgp 3 4
      maximum-paths 10 ecmp 10
      bgp additional-paths install
      bgp additional-paths receive
      bgp additional-paths send limit 2
```